### PR TITLE
Shibboleth integration with Duo support

### PR DESCRIPTION
**Pull request by vacovsky**

### CODE REVIEW

**Comment by vacovsky** [comment]
The package reference for "github.com/vacovsky/saml2aws/pkg/provider/shibboleth" exists but not 	"github.com/versent/saml2aws/pkg/provider/shibboleth" until the PR is merged.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (wolfeidau) [inline .vscode/launch.json]: Can you remove this file please.

File: pkg/provider/shibboleth/shibboleth.go [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+				break
+			}
+		}
+	}
+
+	idpForm := url.Values{}
+	idpForm.Add("_eventId", "proceed")
+	idpForm.Add("sig_response", duoTxCookie+":"+app)
+
+	req, err = http.NewRequest("POST", dpa, strings.NewReader(idpForm.Encode()))
```
Comment (wolfeidau) [inline pkg/provider/shibboleth/shibboleth.go]: You need to check this err.

**Comment by vacovsky** [comment]
Thanks!  Changes made, as requested.

**Comment by cervantek** [comment]
@wolfeidau anything holding this up from getting merged in and making your next release?

**Pull request merged**